index.html

```diff
@@ -287,10 +287,10 @@ <h4><a href="https://code.nasa.gov">Code.nasa.gov</a></h4>
             Congress and only holds standardized metadata about the code.</p>
         <p>Code.nasa.gov only shows open-source software. For government-source
             and patented software, see software.nasa.gov</p>
-        <p class="highlight-red">Please make sure any repos added to github.com/nasa
-            are also tracked in code.nasa.gov!</p>
+        <p class="highlight-red"><strong>Please make sure any repos added to github.com/nasa
```

Comment:
I think this does not have to be placed in direct HTML.
You can put in the class `highlight-red`: `font-weight: bold`